panacea/web/static/js/app.js

```diff
@@ -46,12 +46,15 @@ async function handleDrugsResponse(response) {
       View.displayNoDrugsError();
     }
 
+    // fetching the pml here, TODO: decide when is best to do this
+    const pml = await Request.pml(ast);
```

Comment:
Seeing as the customer didn't buy any PML-TX features. The easiest approach might be to have JS read the file contents on submit.

We could then use that here, and remove the `/api/pml` endpoint entirely.

Any thoughts?